src/applications/lgy/coe/containers/introductionContent/loggedInContent.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+import OMBInfo from '@department-of-veterans-affairs/component-library/OMBInfo';
+
+export const loggedInContent = () => (
+  <>
```

Comment:
The api call for auto COE generation is currently mocked and working, so you should be able to include the markup for it on here per the [mockup](https://preview.uxpin.com/65c0623a799c268173fe1a3cb4375f9ce00ad820#/pages/141257590/simulate/sitemap). 

You might be able to reuse some of the markup from [this component](https://github.com/department-of-veterans-affairs/vets-website/blob/fbf18f18a343385786618e2525f56b42e6b92a9c/src/applications/lgy/coe/components/CertificateDownload.jsx#L0-L1) even though I think it's outdated at this point.

Reply:
We have a separate ticket for the stuff dealing with the COE call. I already combined the un-authed and authed stuff into one ticket so I didn't want to go for a third ticket.